config/defaults.json

```diff
@@ -618,5 +618,25 @@
     "version":        "",
     "parent_key":     "",
     "parent_value":   ""
+  },
+  {
+    "key": "success_center_phone",
```

Comment:
These Settings should be placed in alphabetical order based on key in `defaults.json`.